### DISCUSSION TITLE
Display python objects in sphinx grid

### DISCUSSION THREAD

**Discussion by s-banach** [executablebooks/meta#980] [Q&A]
Putting code cells inside of the grid doesn't seem to work.
Putting the whole grid inside a code cell also doesn't seem to work.
Is there anything else I can try?

**Comment by OriolAbril** [comment]
It is very hard to tell from your question, but I suspect you might not be nesting the elements correctly. The number of backticks or colons in the opening/closing fences should increase the more external a directive is. An example using sphinx-design directives:

```````
``````{grid}
`````{grid-item}
```python
```
`````
``````
```````

The code block is the most internal element, so its fences have 3 backticks, `grid-item` includes the code cell so it needs to have _at least_ 4 backticks, and it has 5 which is OK, similarly `grid` needs _at least_ 6 (one more than `grid` as it needs to include it) and it has 6.
